src/state/probeset.js

```diff
@@ -0,0 +1,15 @@
+import { readable } from 'svelte/store';
+
+// TODO: Make this dynamic based on prod vs local dev.
+const url = '__BASE_DOMAIN__/api/v1/probes/';
```

Comment:
I know you didn't add this comment, but is it necessary? The code here seems fine; it's Rollup that needs to add a production domain.